### PR TITLE
Ks sk975 slownesses translations

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations done / links checked with gitpod

Need Proof for DE, EN and latina versions.

### CODE REVIEW

File: pages/web/hosting/diagnostic_slownesses/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+> Our shared hosting plans do not have a Service Level Agreement (SLA). 
+>
+> If you need a service with an SLA availability rate higher than 99%, we recommend that you consider using a [Virtual Private Server (VPS)](https://www.ovhcloud.com/en-gb/vps/) or [Dedicated Server](https://www.ovhcloud.com/en-gb/bare-metal/).
+>
+> Furthermore, the performance of OVHcloud's shared hosting infrastructure is monitored 24/7. This is to guarantee you high availability, and where applicable, quick recovery of your services in the event of a proven overload.
+>
+
+## Requirements
+
+- Have a website hosted on one of our [OVHcloud shared hosting] offers(https://www.ovhcloud.com/en-gb/web-hosting/)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.en-gb.md]: Link is broken

File: pages/web/hosting/diagnostic_slownesses/guide.es-es.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+> Nuestros planes de hosting no disponen de "Service Level Agreement" (SLA). 
+>
+> Si necesita un servicio con una tasa de disponibilidad SLA superior al 99%, le recomendamos que considere la posibilidad de utilizar un [Servidor Privado Virtual (VPS)](https://www.ovhcloud.com/es-es/vps/) o un [Servidor Dedicado](https://www.ovhcloud.com/es-es/bare-metal/).
+>
+> Por otro lado, el rendimiento de la infraestructura de alojamientos compartidos OVHcloud se monitoriza las 24 horas del día, los 7 días de la semana. Esto le garantiza un alto índice de disponibilidad y, en su caso, una rápida recuperación de sus servicios en caso de sobrecarga comprobada.*
+>
+
+## Requisitos
+
+- Tener un sitio web alojado en uno de nuestros planes de hosting de OVHcloud (https://www.ovhcloud.com/es-es/web-hosting/)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.es-es.md]: link is broken

File: pages/web/hosting/diagnostic_slownesses/guide.it-it.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+>
+> **Dopo aver eseguito tutte le diagnostiche indicate in questa guida**, se il rallentamento viene generato dalla nostra infrastruttura di hosting, ti ricordiamo che è condivisa tra più utenti.
+>
+> Gli utenti condividono le risorse dell'infrastruttura di hosting condivisi per far funzionare i loro siti Web. Se uno di loro è su richiesta dell'infrastruttura condivisa, ciò può avere conseguenze sugli altri hosting presenti sulla stessa infrastruttura.
+>
+> I nostri hosting condivisi non dispongono di "Service Level Agreement" (SLA). 
+>
+> In caso di necessità di un servizio con un tasso di disponibilità SLA superiore al 99%, ti consigliamo di considerare l'utilizzo di un [Server Privato Virtuale (VPS)](https://www.ovhcloud.com/it/vps/) o di un [Server Dedicato](https://www.ovhcloud.com/it/bare-metal/).
+>
+> Inoltre, le prestazioni dell'infrastruttura di hosting condivisi OVHcloud sono monitorate 24 ore su 24 e 7 giorni su 7, al fine di garantirti un alto tasso di disponibilità e, in caso di comprovata necessità, un rapido ripristino dei tuoi servizi.*
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.it-it.md]: please check why there is a single * at the end

File: pages/web/hosting/diagnostic_slownesses/guide.it-it.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+
+Se non vengono segnalati incidenti o interventi di manutenzione, prosegui nella diagnostica.
+
+#### 1.2 - Testare il sito su più dispositivi
+
+Prova il tuo sito Web da un altro dispositivo/computer e poi da un altro punto di accesso a Internet. Svuotare la cache del tuo browser ogni volta che vuoi, in modo che il tuo sito sia caricato direttamente dall'hosting Web.
+
+#### 1.3 - Prova l'hosting con un file indipendente dal tuo sito Web
+
+Nella root del tuo sito Web accedi allo[spazio di storage FTP del tuo hosting condiviso](https://docs.ovh.com/it/hosting/accedere-spazio-storage-ftp-hosting-web/), un file che potrai chiamare `phpinfo.php`.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.it-it.md]:
```suggestion
Nella root del tuo sito Web accedi allo [spazio di storage FTP del tuo hosting condiviso](https://docs.ovh.com/it/hosting/accedere-spazio-storage-ftp-hosting-web/), un file che potrai chiamare `phpinfo.php`.
```

File: pages/web/hosting/diagnostic_slownesses/guide.it-it.md [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+**Caso n°1**
+
+Le seguenti affermazioni si applicano **tutte** alla tua situazione:
+
+- almeno una pagina, uno script o un file (incluso il file "**phpinfo.php**") si è caricato rapidamente durante i test dello step 1;
+- la connessione al tuo database è stata effettuata correttamente durante i test dello Step 1.
+
+> Significa che i ritardi riscontrati provengono direttamente dagli script che compongono il tuo sito Web. È possibile passare direttamente allo Step 2 per seguire i consigli di ottimizzazione per risolvere la tua situazione.
+
+**Caso 2**
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.it-it.md]:
```suggestion
**Caso n°2**
```

File: pages/web/hosting/diagnostic_slownesses/guide.it-it.md [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+
+Le seguenti affermazioni si applicano **tutte** alla tua situazione:
+
+- **nessun incidente e nessuna manutenzione** sono dichiarati o sono stati dichiarati come **risolti** meno di tre ore fa per i tuoi servizi di hosting Web sul nostro sito [status-ovhcloud.com](https://web-cloud.status-ovhcloud.com/);
+- il **cas n°1** dettagliato di cui sopra non corrisponde alla tua configurazione.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.it-it.md]:
```suggestion
- il **caso n°1** dettagliato di cui sopra non corrisponde alla tua configurazione.
```

File: pages/web/hosting/diagnostic_slownesses/guide.it-it.md [hunk 7]
```diff
@@ -0,0 +1,308 @@
...
+
+È possibile scegliere una [offerta di hosting condiviso](https://www.ovhcloud.com/it/web-hosting/) superiore o un'infrastruttura dedicata come [Server Privato Virtuale (VPS)](https://www.ovhcloud.com/it/vps/) o un [Server Dedicato](https://www.ovhcloud.com/it/bare-metal/). 
+
+## Per saperne di più <a name="go-further"></a>
+
+Per prestazioni specializzate (referenziamento, sviluppo, ecc...), contatta i [partner OVHcloud](https://partner.ovhcloud.com/it/).
+
+Per usufruire di un supporto per l'utilizzo e la configurazione delle soluzioni OVHcloud, è possibile consultare le nostre soluzioni [offerte di supporto](https://www.ovhcloud.com/it/support-levels/).
+
+Contatta la nostra Community di utenti all'indirizzo <https://community.ovh.com>.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.it-it.md]:
```suggestion
Contatta la nostra Community di utenti all'indirizzo <https://community.ovh.com/en/>.
```

File: pages/web/hosting/diagnostic_slownesses/guide.pl-pl.md [hunk 8]
```diff
@@ -0,0 +1,309 @@
...
+>Jeśli Twój plik "**index.php**" generuje "powolne" lub ciężkie zapytania do wykonania, TTFB będzie wówczas wysoki, a Twoja strona będzie wyświetlać się kilka sekund. Wydajność Twojego hostingu nie jest zatem istotna.
+
+Narzędzia diagnostyczne online pozwalają pobrać TTFB z Twojego hostingu. Jednak większość z nich działa jak przeglądarki internetowe, więc ich wyniki są relatywistyczne.<br>
+Narzędzia te nie są bowiem w stanie uwzględnić żądań wewnętrznych, o które wystąpił plik wywoływany przez przeglądarkę, jak w powyższym przykładzie z plikiem "**index.php***".
+
+### Etap 1 - Sprawdź, czy opóźnienia wynikają z hostingu lub strony WWW
+
+W tym pierwszym kroku sprawdzisz, czy powstały opóźnienia:
+
+- lub
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.pl-pl.md]: only one word here is strange

File: pages/web/hosting/diagnostic_slownesses/guide.pl-pl.md [hunk 9]
```diff
@@ -0,0 +1,309 @@
...
+
+#### 1.1 - Sprawdź stan usług OVHcloud
+
+Aby mieć pewność, że Twoje usługi (hosting współdzielony **i** baza danych) nie podlegają konserwacji lub awarii, pobierz informacje o klastrze i przędz z hostingu współdzielonego oraz ogólne informacje dotyczące bazy danych. Następnie możesz sprawdzić ich status na stronie [status.ovhcloud.com](https://web-cloud.status-ovhcloud.com/).
+
+Aby dowiedzieć się, gdzie znajduje się Twój hosting, zaloguj się do [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl), przejdź do sekcji `Web Cloud`{.action}, kliknij przycisk `Hosting`{.action} i wybierz odpowiedni hosting. W zakładce `Informacje ogólne`{.action} znajdziesz `centrum` danych hostingu współdzielonego oraz `filer`, w którym znajduje się ono.
+
+![Pobierz Filer](images/DropFilerCluster1.png){.thumbnail}
+
+Następnie kliknij zakładkę `MultiSite{.action}, aby pobrać numer klastra, na którym znajduje się Twój hosting.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.pl-pl.md]:
```suggestion
Następnie kliknij zakładkę `MultiSite`{.action}, aby pobrać numer klastra, na którym znajduje się Twój hosting.
```

File: pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md [hunk 10]
```diff
@@ -0,0 +1,309 @@
...
+> As nossas ofertas de alojamento partilhado não dispõem de "Service Level Agreement" (SLA). 
+>
+> Se precisar de um serviço com uma taxa de disponibilidade SLA superior a 99%, sugerimos que considere a utilização de um [Servidor Privado Virtual (VPS)](https://www.ovhcloud.com/pt/vps/) ou de um [Servidor Dedicado](https://www.ovhcloud.com/pt/bare-metal/).
+>
+> Além disso, as performances da infraestrutura de alojamento partilhado OVHcloud são monitorizadas 24 horas por dia e 7 dias por semana. Isto de forma a garantir-lhe uma alta taxa de disponibilidade e, se for caso disso, um restabelecimento rápido dos seus serviços em caso de sobrecarga comprovada.*
+>
+
+## Requisitos
+
+- Dispor de um site alojado numa das nossas ofertas de[alojamento partilhado OVHcloud](https://www.ovhcloud.com/pt/web-hosting/)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md]:
```suggestion
- Dispor de um site alojado numa das nossas ofertas de [alojamento partilhado OVHcloud](https://www.ovhcloud.com/pt/web-hosting/)
```

File: pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md [hunk 11]
```diff
@@ -0,0 +1,309 @@
...
+Com efeito, estas ferramentas não estão em condições de ter em conta os pedidos internos solicitados pelo ficheiro que ligou através do seu browser, como no exemplo acima com o ficheiro "**index.php**".
+
+### Etapa 1 - determine se as lentidões provêm do alojamento ou do seu website
+
+Esta primeira etapa permite-lhe determinar se os atrasos provêm:
+
+- do seu site, devido ao seu funcionamento interno;
+- ou da infraestrutura de alojamento partilhado onde o seu site se encontra.
+
+Todos os diagnósticos da etapa 1 devem ser realizados ***sem exceção** para determinar se as lentidões provêm dos seus serviços de alojamento web ou do website que aloja.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md]:
```suggestion
Todos os diagnósticos da etapa 1 devem ser realizados **sem exceção** para determinar se as lentidões provêm dos seus serviços de alojamento web ou do website que aloja.
```

File: pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md [hunk 12]
```diff
@@ -0,0 +1,309 @@
...
+
+#### 1.1 - Verifique o estado dos seus serviços OVHcloud
+
+Para ter a certeza de que os seus serviços (alojamento partilhado **e** base de dados) não são objeto de manutenção ou incidente, recupere as informações sobre o cluster e file do seu alojamento partilhado, assim como as informações gerais relativas à sua base de dados. De seguida, poderá verificar o seu estado em [status.ovhcloud.com](https://web-cloud.status-ovhcloud.com/).
+
+Para conhecer o cluster e filer onde se encontra o alojamento partilhado, aceda ao seu [Área de Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt), vá à secção `Web Cloud`{.action}, clique em `Alojamentos`{.action} e escolha o alojamento web em causa. No separador `Information gerais`{.action}, localize o `datacenter` do seu alojamento partilhado bem como o `filer` onde está.
+
+![Recuperar Filer](images/DropFilerCluster1.png){.thumbnail}
+
+A seguir, clique no separador ``Multisites`{.action} para obter o número do cluster onde está o seu alojamento partilhado.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md]:
```suggestion
A seguir, clique no separador `Multisites`{.action} para obter o número do cluster onde está o seu alojamento partilhado.
```

File: pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md [hunk 13]
```diff
@@ -0,0 +1,309 @@
...
+> Para isso, carregue na tecla `F12` quando estiver no seu browser Firefox e selecione o separador `rede`. Carregue a sua página web com os teclados `Ctrl + Maj + R` para que a ferramenta lhe mostrar os pedidos executados para carregar a sua página. Identifique os elementos mais longos a carregar para depois os otimizar.
+>
+>![Análise de rede Firefox](images/F12.png){.thumbnail}
+>
+
+Para diminuir o fluxo de pedidos a cada um dos carregamentos das suas páginas, pode também implementar um Content Delivery Network (CDN). que permitirá colocar em cache o conteúdo estático do seu website. O seu alojamento web será menos solicitado e disporá de mais recursos para tratar o resto dos pedidos que não podem ser colocados em cache.
+
+> [!primary]
+>
+> A OVHcloud oferece várias [ofertas CDN](https://www.ovhcloud.com/pt/web-hosting/options/). Se pretender utilizá-los ou ativar um para o seu alojamento web, aceda à Área de Cliente OVHcloud (https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt) e consulte o nosso guia sobre [utilização do CDN OVHcloud](https://docs.ovh.com/pt/hosting/guia_de_utilizacao_do_acelerador_geocache_num_alojamento_web/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md]: manager link is partially written here

**Review by OVH-Team-Guides — COMMENTED** [review]
Please check all comments for all subs

File: pages/web/hosting/diagnostic_slownesses/guide.en-gb.md (hunk 1 above)
Comment (Kazarus4) [reply pages/web/hosting/diagnostic_slownesses/guide.en-gb.md]: Done

File: pages/web/hosting/diagnostic_slownesses/guide.es-es.md (hunk 2 above)
Comment (Kazarus4) [reply pages/web/hosting/diagnostic_slownesses/guide.es-es.md]: done

File: pages/web/hosting/diagnostic_slownesses/guide.it-it.md (hunk 3 above)
Comment (Kazarus4) [reply pages/web/hosting/diagnostic_slownesses/guide.it-it.md]: done

File: pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md (hunk 13 above)
Comment (Kazarus4) [reply pages/web/hosting/diagnostic_slownesses/guide.pt-pt.md]: done

File: pages/web/hosting/diagnostic_slownesses/guide.pl-pl.md (hunk 8 above)
Comment (Kazarus4) [reply pages/web/hosting/diagnostic_slownesses/guide.pl-pl.md]: done

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**